### PR TITLE
fix: update autonotes default date with new format

### PR DESCRIPTION
**Pull request by ganeshkumar5699**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Update autonotes to use newer default date format, to reduce likelihood of format errors (which can affect other tools which automatically consume release notes)

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: vhdbuilder/release-notes/autonotes/main.go [hunk 1]
```diff
@@ -209,7 +209,7 @@ type flags struct {
 }
 
 var defaultPath = filepath.Join("vhdbuilder", "release-notes")
-var defaultDate = strings.Split(time.Now().Format("2006.01.02 15:04:05"), " ")[0]
+var defaultDate = strings.Split(time.Now().Format("200601.02 15:04:05"), " ")[0]
```
Comment (cameronmeissner) [inline vhdbuilder/release-notes/autonotes/main.go:212]: this is an interesting default date lol

**Review by cameronmeissner — APPROVED** [review]

**Review by anujmaheshwari1 — APPROVED** [review]

File: vhdbuilder/release-notes/autonotes/main.go (hunk 1 above)
Comment (ganeshkumar5699) [reply vhdbuilder/release-notes/autonotes/main.go:212]: it's just for the format

Comment (ganeshkumar5699) [reply vhdbuilder/release-notes/autonotes/main.go:212]: default date would be from time.Now()

File: vhdbuilder/release-notes/autonotes/main.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ func main() {
 	flag.StringVar(&fl.include, "include", "", "only include this list of VHD release notes.")
 	flag.StringVar(&fl.ignore, "ignore", "", "ignore release notes for these VHDs")
 	flag.StringVar(&fl.path, "path", defaultPath, "output path to root of VHD notes")
-	flag.StringVar(&fl.date, "date", defaultDate, "date of VHD build in format YYYY.MM.DD")
+	flag.StringVar(&fl.date, "date", defaultDate, "date of VHD build in format YYYYMM.DD")
```
Comment (alexeldeib) [inline vhdbuilder/release-notes/autonotes/main.go:43]:
```suggestion
	flag.StringVar(&fl.date, "date", defaultDate, "date of VHD build in format YYYYMM.DD.0")
```

I think? or else we need the append?

**Review by alexeldeib — APPROVED** [review]

**Pull request merged**